pkg/broker/work_engine.go

```diff
@@ -29,13 +32,14 @@ type Work interface {
 
 // WorkEngine - a new engine for doing work.
 type WorkEngine struct {
-	topics map[WorkTopic]chan JobMsg
-	bufsz  int
+	subscribers   map[WorkTopic][]WorkSubscriber
+	jobs          map[string]chan JobMsg
```

Comment:
jobs is weird because these aren't the jobs. They're the channels used by the jobs for messages. The name implies the job structs are in this map but that is not the case. Calling `jobs["token"]` will not return me the job I created. Consider renaming it.

Reply:
changed to ``` jobChannels```